src/hotspot/share/opto/traceAutoVectorizationTag.hpp

```diff
@@ -29,25 +29,26 @@
 #include "utilities/stringUtils.hpp"
 
 #define COMPILER_TRACE_AUTO_VECTORIZATION_TAG(flags) \
-  flags(POINTER_PARSING,      "Trace VPointer/MemPointer parsing") \
-  flags(POINTER_ALIASING,     "Trace VPointer/MemPointer aliasing") \
-  flags(POINTER_ADJACENCY,    "Trace VPointer/MemPointer adjacency") \
-  flags(POINTER_OVERLAP,      "Trace VPointer/MemPointer overlap") \
-  flags(PRECONDITIONS,        "Trace VLoop::check_preconditions") \
-  flags(LOOP_ANALYZER,        "Trace VLoopAnalyzer::setup_submodules") \
-  flags(MEMORY_SLICES,        "Trace VLoopMemorySlices") \
-  flags(BODY,                 "Trace VLoopBody") \
-  flags(TYPES,                "Trace VLoopTypes") \
-  flags(POINTERS,             "Trace VLoopPointers") \
-  flags(DEPENDENCY_GRAPH,     "Trace VLoopDependencyGraph") \
-  flags(SW_ADJACENT_MEMOPS,   "Trace SuperWord::find_adjacent_memop_pairs") \
-  flags(SW_REJECTIONS,        "Trace SuperWord rejections (non vectorizations)") \
-  flags(SW_PACKSET,           "Trace SuperWord packset at different stages") \
-  flags(SW_INFO,              "Trace SuperWord info (equivalent to TraceSuperWord)") \
-  flags(SW_VERBOSE,           "Trace SuperWord verbose (all SW tags enabled)") \
-  flags(ALIGN_VECTOR,         "Trace AlignVector") \
-  flags(VTRANSFORM,           "Trace VTransform Graph") \
-  flags(ALL,                  "Trace everything (very verbose)")
+  flags(POINTER_PARSING,            "Trace VPointer/MemPointer parsing") \
```

Comment:
Has anything changed here? I stared at it a few times and could figure out hat has.